### PR TITLE
Update styles on Switch to PAYG

### PR DESCRIPTION
**Pull request by AlexTugarev**
This PR updates styles of the Switch to PAYG page:

**BEFORE**

Screenshot 2023-03-07 at 10 56 52

**AFTER**

Screenshot 2023-03-07 at 10 52 37

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-github-actions
      Experimental feature to run the build with GitHub Actions (and not in Werft).
- [ ] leeway-no-cache
      leeway-target=components:all
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] with-ee-license
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Comment by roboquat** [comment]
@AlexTugarev: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by gtsiolis** [comment]
Looking at this now! 👀

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (gtsiolis) [inline components/dashboard/src/SwitchToPAYG.tsx:444]: **praise**: Thanks for narrowing the max width (`max-w-2xl`) here!

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (gtsiolis) [inline components/dashboard/src/SwitchToPAYG.tsx:451]: **suggestion**: It's worth opening this in a new window, right?

```suggestion
                    href="https://www.gitpod.io/blog/introducing-workspace-classes-and-flexible-pricing"
                    target="_blank"
```

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (gtsiolis) [inline components/dashboard/src/SwitchToPAYG.tsx:481]: **suggestion**: It's worth opening this in a new window, right?

```suggestion
                        <a className="gp-link" href="https://www.gitpod.io/pricing#cost-estimator" target="_blank">
```

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (gtsiolis) [inline components/dashboard/src/SwitchToPAYG.tsx:480]: **thought**: Definitely not the best line to comment on, but I see the button as disabled on the preview environment. I guess this is because I don't have any legacy plans, right?

Comment (gtsiolis) [inline components/dashboard/src/SwitchToPAYG.tsx:480]: **question**: Since we're not making this as a forced flow for users, what do you think it's worth adding a button to skip for now? Thinking that there's no other clear indication, you can bypass this or a cancel button. Button label could be _Skip for now_ or _Reminder me later_. Cc @geropl @atduarte 

| BEFORE | AFTER |
|-|-|
| actions-before | actions-after |

**Review by gtsiolis — APPROVED** [review]
UX LGTM, @AlexTugarev! 🌮 🌮 

Left some minor style comments. I couldn't test the success step.

Approving to unblock merging and holding for discussion or changes.

/hold

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (AlexTugarev) [reply components/dashboard/src/SwitchToPAYG.tsx:422]: Screenshot 2023-03-07 at 12 33 59

Also didn't test it before. There is a missing space before the link.

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (AlexTugarev) [reply components/dashboard/src/SwitchToPAYG.tsx:480]: Exactly.

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (AlexTugarev) [reply components/dashboard/src/SwitchToPAYG.tsx:460]: I think this is only an issue because there is no old subscription. Otherwise you would see an additional line.

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (gtsiolis) [reply components/dashboard/src/SwitchToPAYG.tsx]: Ah, nice. What do you think of replacing the alert with something like this?

Reasoning:
1. Somewhat reward users for making the switch.
2. Describe subtly the benefits they get from the switch.
3. Reuse a familiar pattern for saving progress.

| Success step when switching to PAYG | Project addition | ... |
|-|-|-|
| Screenshot 2023-03-07 at 13 46 10 | Screenshot 2023-03-07 at 13 47 29 | ... |

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (AlexTugarev) [reply components/dashboard/src/SwitchToPAYG.tsx:453]: done

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (AlexTugarev) [reply components/dashboard/src/SwitchToPAYG.tsx]: done

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (AlexTugarev) [reply components/dashboard/src/SwitchToPAYG.tsx]: done

File: components/dashboard/src/SwitchToPAYG.tsx
Comment (AlexTugarev) [reply components/dashboard/src/SwitchToPAYG.tsx]: Awesome!

Comment (AlexTugarev) [reply components/dashboard/src/SwitchToPAYG.tsx]: Screenshot 2023-03-07 at 12 57 13

Last minute change.

**Comment by gtsiolis** [comment]
Thanks, @AlexTugarev!

break-time

**Pull request merged**
